internal/payloads/reservation_payload.go

```diff
@@ -55,7 +55,7 @@ type AWSReservationResponse struct {
 	ID int64 `json:"reservation_id" yaml:"reservation_id"`
 
 	// Pubkey ID.
-	PubkeyID int64 `json:"pubkey_id" yaml:"pubkey_id"`
+	PubkeyID *int64 `json:"pubkey_id,omitempty" yaml:"pubkey_id"`
```

Comment:
```suggestion
	PubkeyID *int64 `json:"pubkey_id,omitempty" yaml:"pubkey_id,omitempty"`
 ```
I guess we should keep yaml and json consistent?

Reply:
Sure, this slipped me through.